CHANGELOG.md

```diff
@@ -6,6 +6,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased 1.2.z]
 
+## [1.2.1] - 2024-11-05
+
+>  No existe una escuela que enseñe a vivir.
+
+### Fixed
+ * Became root after joining an existing user namespace. Otherwise, runc
+   won't have permissions to configure some mounts when running under
+   SELinux and runc is not creating the user namespace. (#4466, #4477)
+
+### Changed
+ * Remove dependency on `golang.org/x/sys/execabs` from go.mod. (#4480)
+ * Remove runc-dmz, that had many limitations, as we are already using
+   other solutions. This was available only in the 1.2.0 release and
+   required to set the environment variable `RUNC_DMZ=true` to opt-in. (#4488)
+ * The script `script/check-config.sh` now checks for overlayfs support. If
+   it is available, it will be used to protect modifications to the runc
+   binary. (#4494)
```

Comment:
This makes an impression that the changes in script/check-config.sh are required for overlayfs protection. They aren't. It's just a convenience script that a user can run to see if their kernel is adequate.

I think something like this will be more to the point:
```suggestion
### Added

* The `script/check-config.sh` script now checks for overlayfs support. (#4494)
```

Reply:
Fixed, thanks